### ISSUE TITLE
Attachments.php returns 404 

### ISSUE THREAD

**Issue by orionrush** [jchristopher/attachments#94]
Scenario:
- In a new post: 
  - Upload an image via Attachments interface 
  - ( _do not 'attach' this image, do not save the post_ )
- In another new post:
  - Navigate to 'Add Media: Media Library' 
  - Find the image previously uploaded 
  - Set Display Settings to 'Attachment page' (the wordpress template)
  - Insert the image into the post
  - Save the post and view it on the front end
  - Follow the link for the inserted image to the it's attachment page
  - Wailing & Gnashing of teeth

In short the image appears in the library, and can be inserted into a post post, but it cannot be linked to a template page as it will return a 404.

Edge case? - probably but it threw me in developing a theme I was testing with a client, as I thought it was something wrong with the template hierarchy.
